src/pages/Admin/License/Create/index.jsx

```diff
@@ -0,0 +1,185 @@
+ version 2 as published by the Free Software Foundation.
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License along
+ with this program; if not, write to the Free Software Foundation, Inc.,
+ 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+*/
+
+import React from "react";
+import InputContainer from "../../../../components/Widgets/Input";
+import Button from "../../../../components/Widgets/Button";
+
+const AddLicense = () => {
+  const handleChange = () => {};
+  const handleSubmit = (e) => {
+    e.preventDefault();
+  };
+  return (
+    <div className="main-container my-3">
+      <div className="row">
+        <div className="col-lg-8 col-md-12 col-sm-12 col-12">
+          <h1 className="font-size-main-heading">Add License</h1>
+          <br />
+          <form>
+            <InputContainer
+              value={""}
+              name=""
+              type="text"
+              id="admin-add-license-status"
+              onChange={handleChange}
+            >
+              Active
+            </InputContainer>
+            <InputContainer
+              value={""}
+              name=""
+              type="text"
+              id="admin-add-license-checked"
+              onChange={handleChange}
+            >
+              Checked
+            </InputContainer>
+            <InputContainer
+              value={""}
+              name=""
+              type="text"
+              id="admin-add-license-spdx-compatible"
+              onChange={handleChange}
+            >
+              SPDX Compatible
+            </InputContainer>
+            <InputContainer
+              value={""}
+              name=""
+              type="text"
+              id="admin-add-license-short-name"
+              placeholder="must be unique"
+              onChange={handleChange}
+            >
+              Short name
+            </InputContainer>
+            <InputContainer
+              value={""}
+              name=""
+              type="text"
+              id="admin-add-license-full-name"
+              onChange={handleChange}
+            >
+              Full name
+            </InputContainer>
+            <div className="my-2">
+              <label htmlFor="upload" className="font-demi font-15">
+                License Text
+              </label>
+              <textarea
+                name="uploadDescription"
+                className="form-control"
+                value={""}
+                id="admin-add-license-text"
+                rows="3"
+                onChange={(e) => handleChange(e)}
+              ></textarea>
+            </div>
+            <InputContainer
+              value={""}
+              name=""
+              type="text"
```

Comment:
Should be a checkbox or Yes/No dropdown.

Reply:
@GMishx I have just added the prototype of add license in it, this pr is getting lot of commits so I thought to do that in next pr. But though now I have added the drop down for it.

Reply:
Okay. But having it correct in the first go will make sure we do not miss it later. Thanks for fixing it :+1: